transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransport.kt

```diff
@@ -0,0 +1,34 @@
+package com.genesys.cloud.messenger.transport.core
+
+import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
+import com.genesys.cloud.messenger.transport.util.DefaultTokenStore
+import com.genesys.cloud.messenger.transport.util.TokenStore
+
+/**
+ * The entry point to the services provided by the transport SDK.
+ */
+class MessengerTransport(configuration: Configuration, private val tokenStore: TokenStore) {
```

Comment:
I'm unsure how you were planning to surface JT. I guess I assumed something like a `createJourneyTrackingClient()` similar to the `createMessagingClient()` that would expose methods to perform JT.

Reply:
Yes, I thought about having something like `createJourneyTrackingClient` function as well. 
But then with current architecture MessengerTransportImpl instance will be created every time client wants to send a JT event, even if there are no need in it. 
Does it makes sense if we move the MessengerTransportImpl instantiation directly to the `createMessagingClient` and remove the field?

Reply:
Perhaps we have differing views of how this would be used. It sounds like you're assuming that users of this library will do something like this for every JT event:

`MessengerTransport(configuration).createJourneyTrackingClient().sendEvent(...)`

Which feels odd. Yes, in the previous `object` based mechanism I guess that was the paradigm. I ran into issues discussed in the group which yielded issues at least on iOS with the object-based mechanism moving forward.

I imagined that the `JourneyTrackingClient` would get injected either by a real DI system or even just via manual constructor injection, so that `MessengerTransport` gets instantiated once, and `JourneyTrackingClient` gets instantiated once, then in the client app they would do:

`journeyTrackingClient.sendEvent(...)`

Reply:
I'll be honest, I can't recall now why I broke it out into the `...Internal` class. I don't see any reason why the implementation can't be in `MessengerTransport`. I wonder if it was an artifact of a failed attempt in a different direction.

Reply:
I've removed `MessengerTransportInternal` entirely and moved the logic into `MessengerTransport`.